safe_price_service/tokens/constants.py

```diff
@@ -0,0 +1,8 @@
+CRYPTO_KITTIES_CONTRACT_ADDRESSES = {
+    "0x06012c8cf97BEaD5deAe237070F9587f8E7A266d",  # Mainnet
+    "0x16baF0dE678E52367adC69fD067E5eDd1D33e3bF",  # Rinkeby
```

Comment:
Rinkeby still makes sense?

Reply:
That `constants `file is not required at all, thanks for noticing